lib/ansible/modules/cloud/ovirt/ovirt_nic.py

```diff
@@ -255,7 +267,12 @@ def main():
             for vnic in connection.system_service().vnic_profiles_service().list():
                 if vnic.name == profile and vnic.network.id == network.id:
                     entitynics_module.vnic_id = vnic.id
-
+        else:
+            # When not specified which vnic use ovirtmgmt/ovirtmgmt
+            networks = networks_service.list()
```

Comment:
Here just check if VM has just single network, if yes, and it has also single VNIC, then set it. If it has multiple networks and user didn't set network fail. If it has single network and multiple VNICs and no vnic specified, fail.

Reply:
Till now it was when creating without profile it didnt failed but created <Empty> profile